### PR TITLE
UserGuide.adoc: Update Undo, Sort, Delete Commands

### LINKED ISSUE #60: Update User guide documentation for sort feature (Week 7)
Update the UG on the list feature by Week 7

### LINKED ISSUE #106: Update Delete feature for UG

### LINKED ISSUE #111: Update undo feature in User Guide
Update undo feature in User Guide

### PR DESCRIPTION
**Pull request by JeremyLoh**
Added information for Undo, Sort, Delete commands.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -170,6 +172,45 @@ Import the contents of the csv file `orders.csv` to Delino.
 The done status of the delivery order will be used to list the delivery orders which have the same status.
 There are three types of list commands that are shown in the examples below.
 
+=== Sorting orders: sort
```
Comment (khsc96) [inline docs/UserGuide.adoc]: the sort here should be `sort` to standardise in TOC

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -170,6 +172,45 @@ Import the contents of the csv file `orders.csv` to Delino.
...
+
+==== Examples
+* `sort` `-a` `n/` +
+You will sort the orders in ascending order based on the names of the customers.
+* `sort` `-d` `ts/` +
+You will sort the orders in descending order based on the timestamps of the orders.
+* `sort` `-d` `tid/` +
+You will sort the orders in descending order based on the transaction id of the orders.
+
+=== Undo the previous action: undo
```
Comment (khsc96) [inline docs/UserGuide.adoc]: here as well

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -170,6 +172,45 @@ Import the contents of the csv file `orders.csv` to Delino.
...
 There are three types of list commands that are shown in the examples below.
 
+=== Sorting orders: sort
+==== Description
+You can sort the orders based on their attributes *(given in prefix form)*.
+An additional flag can be provided (-a or -d) can be used to sort the delivery
+orders in ascending/descending order.
+
+==== Format
+`sort` `[ORDER FLAG]` `ORDER ATTRIBUTE PREFIX` +
```
Comment (khsc96) [inline docs/UserGuide.adoc]: Should we do ORDER_ATTRIBUTE_PREFIX or ORDER ATTRIBUTE PREFIX? I realise @Exeexe93 also doing the no _ format.

**Review by khsc96 — COMMENTED** [review]
Some standardisation issues other than that gj

**Review by khsc96 — APPROVED** [review]
LGTM

**Review by Exeexe93 — APPROVED** [review]
Look good to merge.

**Pull request merged**